include/glow/Importer/ProtobufLoader.h

```diff
@@ -114,6 +114,13 @@ class ProtobufLoader {
   /// if no node has been registered with this name.
   Node *getNodeByNameOrNull(llvm::StringRef name) const;
 
+  /// Create a new variable \p name initialized with \p tensor.
+  /// \returns the newly created variable.
+  Node *createVariable(
```

Comment:
why Tensor &tensor, but not const one? Seems like createVar should not change Tensor values.

Reply:
Good catch! I'd say historical accident :).

Fixing that.